### ISSUE TITLE
Bug in Gradient Accumulation

### ISSUE THREAD

**Issue by raywu0123** [raywu0123/Brain-Tumor-Segmentation#76]
* 不應該把 loss / grad_optim_steps, 應該調小 learning rate
